modules/tanka/Makefile

```diff
@@ -1,6 +1,7 @@
 ## Tanka helpers
 ENVIRONMENTS=$(shell find environments/ -type f -name "main.jsonnet")
 TANKA_EXPORT_FMT?={{.apiVersion}}.{{.kind}}-{{ if.metadata.namespace}}{{.metadata.namespace }}-{{end}}{{.metadata.name }}
+MAKEFILE_DIR=$(shell pwd)
```

Comment:
Should this be BASE_DIR or similar (makefile_root_dir?)

It's just that there's a lot of Makefiles around here ...so which one is this referring to?